iotedgedev/deploymentmanifest.py

```diff
@@ -52,24 +56,58 @@ def add_default_route(self, module_name):
         new_route_name = "{0}ToIoTHub".format(module_name)
         new_route = "FROM /messages/modules/{0}/outputs/* INTO $upstream".format(module_name)
 
-        self.utility.nested_set(self.json, ["moduleContent", "$edgeHub", "properties.desired", "routes", new_route_name], new_route)
+        try:
+            self.utility.nested_set(self.get_module_content(), ["$edgeHub", "properties.desired", "routes", new_route_name], new_route)
```

Comment:
Seems like we can further abstract "get desired property" which takes module and name.

Reply:
@jongio Added. Thanks.